packages/schemas/views/users_with_social_identities.sql

```diff
@@ -0,0 +1,39 @@
+create view users_with_social_identities as
+--- select all columns from users table except identities,
+select
+  u.tenant_id,
+  u.id,
+  u.username,
+  u.primary_email,
+  u.primary_phone,
+  u.password_encrypted,
+  u.password_encryption_method,
+  u.name,
+  u.avatar,
+  u.profile,
+  u.application_id,
+  u.custom_data,
+  u.logto_config,
+  u.mfa_verifications,
+  u.is_suspended,
+  u.last_sign_in_at,
+  u.created_at,
+  u.updated_at,
+  -- custom merged identities column here
+  u.identities || COALESCE(
```

Comment:
is it a temp solution (will remove later)? if not, we have to update multiple places when adding/removing a column in the `user` table. just wanted to double check it's expected.